### DISCUSSION TITLE
Zooz Zen22 OTA FW Update Required Deleting Node, Reinstalling

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [zwave-js/zwave-js-ui#3662] [General]
My first OTA using JSUI was with the Zooz ZEN25 Double Plug (my least critical), which went without a hitch, except it took a couple of re-interviews to finally report the FW was updated from "unknown". Also, all the configuration settings were wrong, and that had to be sorted (it enabled the Auto-Turn Off setting!).

My second OTA was with the ZEN22 Dimmer Switch which was more problematic. It stopped responding to HA/JSUI, and re-interview stalled at first step, and finally reported Dead. I disconnected power to the switch. Nothing. I rebooted my HA host. Nothing. Finally had to remove the node entirely and reinstall it, which renamed it and changed the node. I then had to rename the entities again so that it worked with my dashboard and automations. What a pain.

Anybody else having such issues? I have two other ZEN22s and a couple of other Zooz products that need a FW update, but I'm not sure I want to deal with it.

**Comment by robertsLando** [comment]
Hi @GrizzlyAK , unfortunately the only thing that could help here is to see driver logs. Could you attach them please? 

cc @AlCalzone

**Comment by AlCalzone** [comment]
I've heard from Zooz that some of their devices require to be excluded and re-included after a firmware update. Unfortunately, I haven't heard back from them **which ones** do. We'll add it to the changelog that's shown before updating and work on a way to warn users before the update.

**Comment by GrizzlyAK** [comment]
Thanks guys. It sounds like this is a Zooz issue and not ZWJS. I think a warning for Zooz devices would be good to let users know what to expect, even if generic for all Zooz devices. Although I had seen the scrambling/reordering of config parameters in early releases of some Zooz products, I had forgotten about it and is something else users should be aware of when doing OTA FW upgrades. I think sometimes the changes are warranted, especially when new features are added, but I think none of the changes I saw on the Double Plug were necessary, just a reordering of params and resetting values which broke everything. That didn't seem necessary. Zooz should be made aware that changing this stuff willy-nilly has consequences beyond their convenience. Maybe explain to them the results are severely exchanged functionality, rewritten automations, etc. If this plug had been controlling power to a freezer/refrig, for example... which 3 of my 9 ZEN15s actually do (and they have auto-off settings too).

Also, to follow up, I ultimately had to do a LOT of finagling with the Plug to get it work. Many reinterviews, cycle power, etc. I didn't have to remove it (yet), so it seems to be working again.

The actual OTA update process from a ZWJS standpoint, though, was fantastic!
